migrations/sql.go

```diff
@@ -28,6 +28,7 @@ var PostgresFieldTypes map[proto.Type]string = map[proto.Type]string{
 	proto.Type_TYPE_SECRET:    "TEXT",
 	proto.Type_TYPE_PASSWORD:  "TEXT",
 	proto.Type_TYPE_MARKDOWN:  "TEXT",
+	proto.Type_TYPE_VECTOR:    "VECTOR(1536)",
```

Comment:
```suggestion
	proto.Type_TYPE_VECTOR:    "VECTOR",
```

I think passing the size will prevent the field from being used with anything other than 1536 dimensions in this case!?